datafusion/common/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+pub mod alias;
```

Comment:
Need suggestion on the best place to put the AliasGenerator code. Currently, I put it in common because it is used in `core`, `optimizer`, and `physical-expr` crate.

Reply:
I think putting it in `common` is a reasonable location 👍